luatest/replica_set.lua

```diff
@@ -0,0 +1,231 @@
+--- Class to manage groups of Tarantool instances with the same data set.
+--
+-- @classmod luatest.replica_set
+
+local checks = require('checks')
+local log = require('log')
+
+local helpers = require('luatest.helpers')
+local Server = require('luatest.server')
+
+local ReplicaSet = {}
+
+function ReplicaSet:inherit(object)
+    setmetatable(object, self)
+    self.__index = self
+    return object
+end
+
+--- Build a replica set object.
+--
+-- @tab[opt] object
+-- @tab[opt] object.servers List of server configurations to build server
```

Comment:
maybe, `server_configs`? The name `servers` is already used by internal field with a table of `luatest.server` objects.